README.md

```diff
@@ -2,152 +2,107 @@
 
 This repository contains the official Account Abstraction - Hybrid Compute snap.
 
-## Audit
-The snap has been audited by Sayfer, [report](https://sayfer.io/audits/metamask-snap-audit-report-for-enya-labs/)
+[Hybrid Compute](https://docs.boba.network/hc) utilizes [Account Abstraction](https://docs.boba.network/developer/features/aa-basics) on Boba Network to engage with off-chain APIs seamlessly. Account Abstraction authenticates your transactions with ECDSA private keys (as well as lowering your costs by bundling your transactions). Hybrid Compute enables you to engage with GenAI or any other off-chain data, increasing the power and interoperability of your MetaMask.
+
+Unlock the power of real-world data for Web3 with the Account Abstraction - Hybrid Compute snap; a single line of code is all it takes.
+
+## Install the Snap
+
+To install the snap, follow [this link](https://hc-wallet.sepolia.boba.network/) and select "Install Snap & Connect." This will also connect your MetaMask to the Boba Sepolia Network. You can then create your "AA HC Wallet" account; we recommend creating it deterministically, though you have the option to import a private key instead. If you choose that option, *please* ensure you back up and save your private key.
```

Comment:
```suggestion
To install the snap, follow [this link (only Boba Sepolia as of 21 Nov 2024)](https://hc-wallet.sepolia.boba.network/) and select "Install Snap & Connect." This will also connect your MetaMask to the Boba Sepolia Network. You can then create your "AA HC Wallet" account; we recommend creating it deterministically, though you have the option to import a private key instead. If you choose that option, *please* ensure you back up and save your private key.
```